nomad/local/nomad_cli_tools.sh

```diff
@@ -21,36 +21,39 @@ nomad_dispatch() {
     echo "${job_id}"
 }
 
+url_to_nomad_job_in_ui() {
+    local job_id="${1}"
+    # urlencode
+    job_id=$(jq -rn --arg input "${job_id}" '$input|@uri')
```

Comment:
It might be nice to rename this var to urlencoded_job_id to clarify what this var is expecting